components/messages/README.md

```diff
@@ -0,0 +1,123 @@
+# Messsages
+
+This component manager the messages.
+
+## Routes
+
+```http
+/api/messsages
+```
+
+### POST `/api/messsages`
+
+Create a message record.
+
+Data send in body as JSON.
+
+```js
+{
+  "patientId": "",
+  "messageText": ""
+}
+```
+
+Response code 201:
+
+```js
+{
+  "data": createMessageId,
+  "message": "Message created"
+}
+```
+
+Response code 404:
+
+```js
+{
+    "statusCode": 404,
+    "error": "Not Found",
+    "message": "Not Found",
+}
+```
```

Comment:
i don't find not found error. this is not a possible response.

Reply:
fixed